pages/common/pssh.md

```diff
@@ -1,27 +1,27 @@
 # pssh
 
-> Parallel ssh program.
+> Parallel SSH program.
 
-- Run a command on host1 and host2 to print the date on each server inline:
+- Run a command on two hosts, and print its output on each server inline:
 
-`pssh -i -H "{{host1}} {{host2}}" {{date}}`
+`pssh -i -H "{{host1}} {{host2}}" {{echo "foo"}}`
 
 - Run a command and save the output to separate files:
 
-`pssh -H {{host1}} -H {{host2}} -o {{path/to/output_dir}} {{date}}`
+`pssh -H {{host1}} -H {{host2}} -o {{path/to/output_dir}} {{echo "foo"}}`
 
-- Run a command on each host specified in a new-line separated host file:
+- Run a command on multiple hosts, specified in a new-line separated file:
 
-`pssh -i -h {{path/to/hosts_file}} {{date}}`
+`pssh -i -h {{path/to/hosts_file}} {{echo "foo"}}`
 
-- Run a command as root; asks for root password:
+- Run a command as root (this asks for the root password):
 
-`pssh -i -h {{path/to/hosts_file}} -A -l {{root}} {{date}}`
+`pssh -i -h {{path/to/hosts_file}} -A -l {{root_username}} {{echo "foo"}}`
 
 - Run a command with extra SSH arguments:
 
-`pssh -i -h {{path/to/hosts_file}} -x "{{-O VisualHostKey=yes}}" {{date}}`
+`pssh -i -h {{path/to/hosts_file}} -x "{{-O VisualHostKey=yes}}" {{echo "foo"}}`
 
-- Run a command using a maximum number of concurrent connections:
+- Run a command using the maximum number of concurrent connections:
```

Comment:
Perhaps "specifying" here?

Reply:
Ah, that does make it clearer. IIUC, then, the limit is set by the user, otherwise the default is to use as many as possible -- is that correct?

Reply:
I think so, yeah

Reply:
@deekim please confirm, just to be extra sure :)

Reply:
http://www.theether.org/pssh/docs/0.2.3/pssh-HOWTO.html
> By default, pssh uses at most 32 ssh processes in parallel to ssh to the various nodes.

32 i believe.